### DISCUSSION TITLE
Resource Owner Password Flow?

### DISCUSSION THREAD

**Discussion by shrike71** [fief-dev/fief#10] [Ideas]
Hi @frankie567 ,

Any chance if fief supports the Resource Owner Password Flow? I know the default is Authorization Code, but was wondering if this is also supported and any chance of a quick Flask or FastAPI example?

Authorization flow working famously with custom domains from a local install, BTW

Thanks!

**Comment by frankie567** [comment]
Right now, it's not implemented; but that could be something interesting to have indeed. Wondering what would be the implications (if any) in terms of security 🤔 

EDIT: I have in mind to let developers implement their own authentication pages if they don't want to use the one provided by Fief. So that could be definitely something helpful for this purpose.
